pkg/tweetnacl/Makefile

```diff
@@ -0,0 +1,11 @@
+PKG_NAME=tweetnacl
+PKG_URL=https://github.com/kYc0o/tweetnacl
+PKG_VERSION=3548e2c0474731123e8e22a8049a945cd227aea6
+
+.PHONY: all
+
+all: git-download
+	@cp Makefile.tweetnacl $(PKG_BUILDDIR)/Makefile
```

Comment:
I think adding a makefile with explicit RIOT stuff would avoid to use the sources for other purposes than on RIOT, assuming that such purposes would exist...